challenge_01/style.css

```diff
@@ -0,0 +1,95 @@
+body {
+    background-color: #E2F1E7;
+    font-family: "Figtree", sans-serif;
+    font-size: 9px;
+}
+
+.form-wrapper {
+    background-color: #ffffff;
+    width: 400px;
```

Comment:
1. Better to use `inline-size` for width
2. Better to use something dynamic like max-width or according to point one, max-inline-size

Reply:
> 1. Better to use `inline-size` for width
> 2. Better to use something dynamic like max-width or according to point one, max-inline-size

I don't get this 🤔

Reply:
I was trying to point to two different things and it made it confusing.

First thing: use logical properties where ever possible inline-* and block-* properties
Second completely different thing: keep dynamic styling in mind which is to use max-* and min-* sizes for width and height and other things. If you set the `width` to `400px` it will make a problem when the screen size goes below `400px` so you have to use media queries and stuff to handle those cases but with having `max-inline-size` set to `400px` the card will shrink when the screen size goes below `400px` and you won't need heavy styling in media queries.

Result from 1 and 2: `max-inline-size: 400px;`

Reply:
Got it.
Thanks 🤝